.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,23 +1,28 @@
 name: 🐛 | Bug Report
-description: >
-  File a bug report
+description: Submit a bug report
 title: "[🐛 | Bug]: "
 labels:
   - bug
   - triage
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping improve Parsec! Please fill in the required fields below and add any extra
+        information that can help us understand the issue.
   - type: input
     id: version
     attributes:
-      label: Version
-      description: What version of our software are you running ?
-      placeholder: version
+      label: Parsec version tested on:
+      description: You can find it in Parsec GUI title bar (or by running `parsec.cli --version` in a terminal).
+      placeholder: 2.15.0
```

Comment:
Need to be managed by `version_updater` :thinking:

Reply:
Actually it would be nicer to have a `dropdown` becase we do not need to support every single Parsec version. If the problem is found in a pre-release, intermediate commit, or something then it is an internal issue and not a (public) bug.

- 2.15 (stable)
- 2.16 (edge)
- Older version (specify below)
- master branch

Reply:
Having a dropdown would be nice but it would easily be forgotten when doing a release :sweat_smile: